modules/ROOT/pages/install-checklist.adoc

```diff
@@ -30,6 +30,7 @@ Installing Anypoint Platform PCE requires specific versions of operating systems
 === Environment Prerequisites
```

Comment:
In the first line where it says "Before you install Anypoint Platform PCE,...", the "you" could be misleading since it might imply that the customer is able to install PCE by himself when they actually need services to install it